docs/docs/plugins-themes-and-starters.md

```diff
@@ -37,6 +37,12 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
 - [Creating a starter](/docs/creating-a-starter/)
 - [Converting a starter to a theme](/docs/themes/converting-a-starter/)
 
+## Conventions for usage
+
+Themes are a type of plugin, making both themes and plugins capable of the same functionality. The difference between them is the _intent for their usage_. Themes are intended to own a piece of the site (like an about us page), whereas a plugin is intended to modularize Gatsby APIs into smaller pieces. Themes tend to cover a broader scope of responsibility, packaging up multiple behaviors, where plugins are meant for a narrower use.
```

Comment:
```suggestion
Themes are a type of plugin, making both themes and plugins capable of the same functionality. The difference between them is _intented usage_. Themes are intended to own a piece of the site (like an about us page), whereas a plugin is intended to modularize Gatsby APIs into smaller pieces. Themes tend to cover a broader scope of responsibility, packaging up multiple behaviors, where plugins are meant for more specific functionality.
```
Not sold on my edit for the last sentence. I know what we're trying to say, but it's tough to explain.